manifests/base/config/argocd-ssh-known-hosts-cm.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/name: argocd-ssh-known-hosts-cm
+    app.kubernetes.io/part-of: argocd
+  name: argocd-ssh-known-hosts-cm
+data:
+  ssh_known_hosts: |
```

Comment:
I checked these once. Should check again.

Reply:
I took the data verbatim from `hack/ssh_known_hosts`, you can probably C&P from the CM data, trim the leading white space, and diff against the file. But that brings me to another question: For declarative setup, it might be benificial to just deliver a CM with blank data, and load the known hosts data initially after first installation, such as
`argocd cert add-ssh --batch --from hack/ssh_known_hosts`
Otherwise I guess, the data in the CM could be overwritten each time a user re-applies the manifest, e.g. at update time. What do you think?